build-parent/pom.xml

```diff
@@ -1237,6 +1238,8 @@
             <properties>
                 <!-- Podman compatibility. Currently, mac file systems based on Plan 9 does not support SELinux labeling z and Z should not be used. When they transition to use virtiofsd, it should support SELinux labeling, and then we can use it for better container separation on the Mac.-->
                 <volume.access.modifier></volume.access.modifier>
+                <!-- An ARM image for db2 isn't available, but if we request the amd64 one it all works fine -->
+                <db2.image.platform>linux/amd64</db2.image.platform>
```

Comment:
Interesting :)
But why is the default for `db2.image.platform` set to `${os.family}/${os.arch}` then - do we support other arch's ?
I'm wondering if it shouldn't be hardcoded to `linux/amd64` for all cases.

Reply:
No no no no please ignore this PR @Sanne :)

Reply:
(I just chose any old branch to do a PR against a local repo, and apparently `%dev.quarkus-bot.dry-run=true` doesn't do what I think ... )

Reply:
... but, to answer your question, it's an M1 podman thing. For some Docker images, with Podman, you need to explicitly set the platform you want to run on. I don't fully understand why you sometimes need this and sometimes don't, but I guess it's a detail of the multi-arch support in Podman and also in the image being pulled. 

So we don't want to always request `linux/amd64`, because normally we want to let the tool defaults do their job - and especially if an aarch64 image is available, we want to pull it on aarch64.  But for DB2, we have to specifically set the platform, and we have to set it to a platform we're not.

Reply:
... and to *actually* answer your "I'm wondering if it shouldn't be hardcoded to linux/amd64" question, at the moment I think you're right that we only support two architectures, amd/intel, which needs the arch to be set to `linux/amd64`, and M1, which needs the arch to be set to `linux/amd64`. So hardcoding would work, but it just felt wrong. We'd also need to hunt down and undo the hardcoding if we wanted to build on something like linux on raspberry pi, which isn't inconceivable.

Reply:
Got it. Thanks!